.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+yarn-error.log
+yarn.lock
```

Comment:
`yarn.lock` should not be gitignored - it helps ensure that everyone is running the same version of each dependency in their development environment